build.conf

```diff
@@ -22,6 +22,7 @@ paths =
   crypto/crypt_blowfish.c
   dbm/apr_dbm_sdbm.c
   dbm/apr_dbm.c
+  dbm/apr_dbm_lmdb.c
```

Comment:
This should be removed, the DBM drivers which can be built as DSOs should not be listed here otherwise it will get linked twice.